### ISSUE TITLE
make used model part of return payload

### ISSUE THREAD

**Issue by amn41** [RasaHQ/rasa_nlu#854]
feature suggestion: include the model that was used to predict in the payload: 

(this is especially relevant since we introduced projects and you want to sanity check you are using the latest model)

e.g.: 
```
{
  "entities": [], 
  "intent": {
    "confidence": 0.876606527123081, 
    "name": "greet"
  }, 
  "text": "hello", 
  "intent_ranking": [
    {
      "confidence": 0.876606527123081, 
      "name": "greet"
    }, 
    {
      "confidence": 0.13460273299354185, 
      "name": "goodbye"
    }, 
  ],
  "project": "default",
  "model": "model_20180202-150000"
}
```

**Comment by RitwikGopi** [comment]
I would like to do this. This would be my first ever proper contribution to an open source project. So can you please guide me a bit. Should I make this change as a different branch or on master itself? Can I make this change by creating a branch from tag 0.12.0? I am not able to run the code at current master and test the results.

**Comment by RitwikGopi** [comment]
I have made this change and created a pull request. Please let me know if I need to make any changes.

**Comment by wrathagom** [comment]
Closed by #1019
